### ISSUE TITLE
[QUESTION] <Brief Description>Distillation dinov3 to yolov5, loss dose not decrease

### ISSUE THREAD

**Issue by YingdiZhang** [lightly-ai/lightly-train#397]
### 🤔 What’s your question?

Thanks for the great job. When I distill dinov3 to yolov5, I found the loss dose not decrease

### 🧠 What did you try?

Explain what you’ve already looked into or attempted.

### 📌 Related code or context (if any)

import lightly_train
from ultralytics import YOLO

if __name__ == "__main__":
    model = YOLO("yolov5s.yaml")

    lightly_train.train(
        out="distill_exp/dinov3_vitb16_2yolov5s",  
        data="/data/data_zyd/foundation_model_data",
        model=model,
        method="distillationv2",
        accelerator="gpu",
        devices=[0,1,2,4,5,6,7],
        epochs=500, 
        batch_size=7168, 
        method_args={
            "teacher": "dinov3/vitb16", 
            # Replace with your own url
            "teacher_url": "/data/zyd/dinov3_pth/dinov3_vitb16_pretrain_lvd1689m-73cec8be.pth"
        }
    )

```python
# Related snippet if relevant
```

**Comment by stegmuel** [comment]
Hi,

DINOv3 models  seem to work better with `distillationv1`. 
I would also recommend you to use batch sizes in the range [1024-2048].

**Comment by YingdiZhang** [comment]
Thansk for your reply. When using distillationv1 and setting batch_size=2048, should I change other hyperparameters such as learning rate ?

**Comment by stegmuel** [comment]
No, the learning rate is automatically scaled to the batch size and I think the resulting value should be a good start.
